ui/src/app/pages/signed-in/access-renewal-notification.tsx

```diff
@@ -37,6 +43,18 @@ export const AccessRenewalNotificationMaybe = (
       ? daysRemaining + ' days remaining.'
       : 'Your access has expired.';
 
+  const boxColor = cond(
+    [twoWeeksOrMoreRemaining, () => colors.notificationBanner.EXPIRING_SOON],
```

Comment:
Would have like to return the style here itself but that isnt working. Suggestion welcome

Reply:
I think this is fine.

Reply:
Returning the values is fine with me too.

But I would like to see the cases in the conds match, to make it more obvious how they go together.  One way to do this is to combine them:
```
const [boxColor, iconColor] = cond(...
```
There's an example of this pattern at `pause-resume-button`